cli/flox/src/commands/containerize.rs

```diff
@@ -50,7 +68,15 @@ impl Containerize {
         };
 
         let (output, output_name): (Box<dyn Write + Send>, String) =
-            if output_path == Path::new("-") {
+            if let Some(registry) = self.load_into_registry {
+                let command = Command::new(&registry)
+                    .arg("load")
+                    .stdin(Stdio::piped())
+                    .spawn()
+                    .context("Could not start registry load command")?;
+
+                (Box::new(command.stdin.unwrap()), registry.to_string())
+            } else if output_path == Path::new("-") {
```

Comment:
blocking to discuss: is it worth adding a test to `containerize.bats`?

Reply:
Oh, certainly, just wanted to know if the way I did this was okay. Sorry for not following a pure TDD flow. 😅

Reply:
That makes is sound like you think we follow a TDD flow lol